src/components/FeatureViews/Courses/CourseClasses.js

```diff
@@ -215,6 +224,17 @@ const CourseClasses = () => {
 
   const handleChange = (_, i) => setIndex(i);
 
+  const accountTypeCheck = (accountType, studentList, enrollmentArray) => {
```

Comment:
is this to return the tabs depending on the account type? if so rename to describe that: "setTabsForAccountTypes()`